src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -66,6 +66,33 @@ class BasicWorkflow extends FunSpec with Matchers {
         }
 
         //-----------------------------------------------------------------------------------------------
+        // public symbol search - java.io.File
+
+        val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
+        javaSearchSymbol match {
+          case SymbolSearchResults(List(
+            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
+              Some(_)),
+            TypeSearchResult("java.io.File", "File", 'class,
+              Some(_)),
+            TypeSearchResult("org.ensime.util.FileUtilSpec", "FileUtilSpec", 'class, None))) =>
```

Comment:
1. that looks like a bug. The max number must not be going through to lucene. Also see #637 
2. this is actually reasonable: "io" will hit the `ImageObserver` via CamelCase... but `java.io.File` should be a much stronger hit.